packages/context/src/context-view.ts

```diff
@@ -161,10 +161,10 @@ export class ContextView<T = unknown> extends EventEmitter
  */
 export function createViewGetter<T = unknown>(
   ctx: Context,
-  bindingFilter: BindingFilter<T>,
+  bindingFilter: BindingFilter,
```

Comment:
Before this change, it was possible to infer `T` in `createViewGetter` from `bindingFilter` type.

```ts
const filter: BindingFilter<MyValue> = b => true;
const getter = createViewGetter(ctx, filter);
// ^^^ getter is of type Getter<MyValue[]>
```

With this change in place, users have to explicitly specify the getter type, otherwise they end up with `unknown`.

```ts
const getter = createViewGetter(ctx, filter);
// ^^^ getter is of type Getter<unknown[]>

const getter = createViewGetter<MyValue>(ctx, filter);
// ^^^ getter is of type Getter<MyValue[]>
```